database_cleanup/models/purge_data.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# © 2014-2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import _, api, fields, models
+from openerp.exceptions import UserError
+
+
+class CleanupPurgeLineData(models.TransientModel):
+    _inherit = 'cleanup.purge.line'
+    _name = 'cleanup.purge.line.data'
+
+    data_id = fields.Many2one('ir.model.data', 'Data entry')
+    wizard_id = fields.Many2one(
+        'cleanup.purge.wizard.data', 'Purge Wizard', readonly=True)
+
+    @api.multi
+    def purge(self):
+        """
+        Unlink data entries upon manual confirmation.
```

Comment:
Use docstring without empty-newline in the first line reference: https://www.python.org/dev/peps/pep-0257/#specification